oem/smc/drive.go

```diff
@@ -0,0 +1,79 @@
+package smc
```

Comment:
Oops, need to include the copyright header:

```go
//
// SPDX-License-Identifier: BSD-3-Clause
//

package smc
```

Reply:
Thanks!